src/index.html

```diff
@@ -8,9 +8,41 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="bigSearch">
+      <form
+        type="text"
+        method="get"
+        data-qa="big"
+        action="/"
+        class="bigSearchForm"
+      >
+        <input
+          type="search"
+          class="bigSearchInput searchInput"
```

Comment:
better add all style for big input to searchInput class and change style for small input (can use BEM and call small input - searchInput--small